apps/mobile/src/components/Community/CommunityView.tsx

```diff
@@ -44,7 +69,63 @@ export function CommunityView({ queryRef }: Props) {
   if (!community || community.__typename !== 'Community') {
     throw new Error(`Unable to fetch the community`);
   }
-  const { contractAddress } = community;
+
+  const navigation = useNavigation<MainTabStackNavigatorProp>();
+
+  const handleUsernamePress = useCallback(() => {
+    if (community.creator?.__typename === 'GalleryUser' && community.creator?.username) {
+      navigation.navigate('Profile', { username: community.creator?.username });
+    }
+  }, [community.creator, navigation]);
+
+  const totalTokens = community.tokensInCommunity?.pageInfo?.total;
+
+  const formattedTotalTokens = useMemo(() => {
+    if (totalTokens && totalTokens > 999) {
+      return `${Math.floor(totalTokens / 1000)}K`;
+    }
+
+    return totalTokens;
+  }, [totalTokens]);
+
+  const showAddressOrGalleryUser = useMemo(() => {
+    if (community.creator?.__typename === 'GalleryUser' && !community.creator?.universal) {
+      return (
+        <GalleryTouchableOpacity
+          className="flex flex-row items-center space-x-1"
+          onPress={handleUsernamePress}
+          eventElementId="NFT Detail Token Owner Username"
+          eventName="NFT Detail Token Owner Username"
+        >
+          {community.creator.__typename && <ProfilePicture userRef={community.creator} size="xs" />}
+
+          <Typography
+            className="text-sm text-shadow"
+            font={{ family: 'ABCDiatype', weight: 'Regular' }}
+          >
+            {community.creator.username}
+          </Typography>
+        </GalleryTouchableOpacity>
+      );
+    } else if (community.contractAddress) {
+      return (
+        <View className="flex flex-row space-x-1 items-center">
+          <RawProfilePicture
+            size="xs"
+            default
+            eventElementId="ProfilePicture"
```

Comment:
maybe null here for eventElementId and eventName since pressing doesn't do anything

Reply:
if null isn't allowed, maybe make it more specific like "Community Page Creator Profile Picture"